src/components/Rating/Rating.js

```diff
@@ -75,34 +98,23 @@ export default class Rating extends Component {
       )
     }
 
-    return (
-      <div
-        style={this.props.disabled || this.props.readOnly ? {...styles.disabled, ...this.props.style} : this.props.style}
-      >
-        {rating}
-      </div>
-    )
+    return (<div className={this.props.classes.root}>{rating}</div>)
   }
 }
 
 Rating.defaultProps = {
   disabled: false,
-  iconFilled: <Star color={orange[500]} />,
-  iconHovered: <StarBorder color={orange[500]} />,
-  iconNormal: <StarBorder color={grey[300]} />,
   tooltipPosition: 'bottom-center',
   max: 5,
   readOnly: false,
   value: 0
 }
 
 Rating.propTypes = {
-  /** Sets classname for IconButton component. */
-  className: PropTypes.string,
+  /** Sets classes for IconButton and SvgIcon components. */
```

Comment:
:+1: for documenting the new props, would you mind adding the supported classes? Material-UI also documents them for every component

Reply:
Done